src/types/websockets/client.ts

```diff
@@ -1,3 +1,6 @@
+import { AxiosRequestConfig } from 'axios';
+import { RestClientOptions } from 'lib/requestUtils';
```

Comment:
```suggestion
import { RestClientOptions } from './lib/requestUtils';
```

not sure why vscode is doing this by default (you're not the first one to see this). Relative path needed or the build won't work